src/UI/Core/UIContainer.cpp

```diff
@@ -35,27 +35,27 @@ UIContainer::~UIContainer()
 /**
  * Adds a Control to the UIContainer.
  */
-void UIContainer::add(Control* c, float x, float y)
+void UIContainer::add(Control* control, float x, float y)
```

Comment:
Another PR changing the signature to `add(Control* control, Point<float> position)` would be helpful

Reply:
Agreed.

I would also like to change the first parameter to `Control&`, since it is non-nullable. That affects a lot of calling code though.

Reply:
Actually, now that I think about it, a `Vector<float>` might be a good thing to add as well:

```
add(Control* control, Point<float> absolutePosition);
add(Control* control, Vector<float> normalizedPosition, Vector<float> offset);
```